sky/skylet/autostop_lib.py

```diff
@@ -57,6 +59,19 @@ def set_autostop(idle_minutes: int, backend: Optional[str], down: bool) -> None:
         set_last_active_time_to_now()
 
 
+def set_is_autostopping() -> None:
```

Comment:
I thought it's a bool, because I read
* `autostop_lib.set_is_autostopping()`
* "is_autostopping flag" in docstrs here
* `_AUTOSTOP_INDICATOR`

which suggest it's a bool.

But in these impls it's a timestamp.

How about we change these to be consistent? E.g., `set_autostopping_timestamp()`, update docstrs, etc.

Reply:
Good point! I renamed it to `set_autostopping_indicator` and update the comments. How does it look?